modules/ip6/cli/icmp6.c

```diff
@@ -196,10 +211,14 @@ static int ctx_init(struct ec_node *root) {
 		CLI_CONTEXT(
 			root, CTX_ARG("ping", "Send ICMPv6 echo requests and wait for replies.")
 		),
-		"DEST [vrf VRF] [count COUNT] [delay DELAY]",
+		"DEST [vrf VRF] [count COUNT] [delay DELAY] [source NAME]",
```

Comment:
Could you rename this parameter to `[iface IFACE]` to go with my previous comment?